### PR TITLE
FEATURE: Add gradient-container atom

### PR DESCRIPTION
**Pull request by c4ll-m3-j4ck**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     theme: {
         gradients: {
             blue: '#26224C 0%, #0114C2 100%',
+            blueDouble: '#26224C 0%, #0114C2 50%, #26224C 100%',
```
Comment (jonnitto) [inline tailwind.config.js:13]:
```suggestion
            blue-inside: '#26224C 0%, #0114C2 50%, #26224C 100%',
```

File: DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Atom/GradientContainer.fusion [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        colorClass = Neos.Fusion:Case {
+            blueDouble {
+                condition = ${props.variant == 'blueDouble'}
+                renderer = 'gradient-blueDouble'
+            }
+            default {
+                condition = ${true}
+                renderer = 'gradient-blue'
+            }
+        }
```
Comment (jonnitto) [inline DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Atom/GradientContainer.fusion:20]:
```suggestion
        colorClass = Neos.Fusion:Match {
            @subject = ${props.variant}
            @default = 'gradient-blue'
            blue-inside = 'gradient-blue-inside'
        }
```

File: DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Atom/GradientContainer.fusion [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        directionClass = Neos.Fusion:Case {
+            bottom {
+                condition = ${props.direction == 'bottom'}
+                renderer = 'bg-gradient-to-b'
+            }
+            default {
+                condition = ${true}
+                renderer = 'bg-gradient-to-r'
+            }
+        }
```
Comment (jonnitto) [inline DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Atom/GradientContainer.fusion:31]:
```suggestion
        directionClass = Neos.Fusion:Match {
            @subject = ${props.direction}
            @default = 'bg-gradient-to-r'
            bottom = 'bg-gradient-to-b'
        }
```

File: DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Atom/GradientContainer.fusion [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    @styleguide {
```
Comment (jonnitto) [inline DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Atom/GradientContainer.fusion:34]: `@styleguide` to the top of the file

File: DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Atom/GradientContainer.fusion [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        title = 'Gradient Container'
+        props {
+            direction = 'bottom'
+            variant = 'blue'
+            content = Neos.Fusion:Tag {
+                attributes.class = 'h-64'
+            }
```
Comment (jonnitto) [inline DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Atom/GradientContainer.fusion:41]:
```suggestion
            content = afx`<div class="h-64"></div>`
```

**Review by jonnitto — CHANGES_REQUESTED** [review]

**Comment by c4ll-m3-j4ck** [comment]
After clarifying the requirements with @manuelmeister we have agreed to not use this component. I'm closing this PR since it is no longer needed!

**Pull request closed without merging**